tests/docker-images/statestore-image/image_builder.sh

```diff
@@ -18,22 +18,34 @@
 # * limitations under the License.
 # */
 
+IMAGE_NAME=apachebookkeeper/bookkeeper-current:latest
+if [[ "$(docker images -q $IMAGE_NAME 2> /dev/null)" != "" ]]; then
+  echo "reusing local image: $IMAGE_NAME"
+  exit 0
```

Comment:
One problem I see with this optimization is that, when debugging or locally running test, I would not know for sometime why changes I made is not taking affect, until I see that image_builder is not building new image anymore.

Reply:
I added a env property to force the rebuilding of the images. By default is false since it's uncommon you want to refresh those images